src/NodeApi/Runtime/NodeEmbeddingPlatformSettings.cs

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.JavaScript.NodeApi.Runtime;
+
+using static NodeEmbedding;
+using static NodejsRuntime;
+
+public class NodeEmbeddingPlatformSettings
+{
+    public NodeEmbeddingPlatformFlags? PlatformFlags { get; set; }
+    public string[]? Args { get; set; }
+    public ConfigurePlatformCallback? ConfigurePlatform { get; set; }
+
+    public static JSRuntime JSRuntime => NodeEmbedding.JSRuntime;
```

Comment:
nit: Delete this static property and reference `NodeEmbedding.JSRuntime` instead on line 22 below.